src/Containers/Chair.tsx

```diff
@@ -10,65 +13,76 @@ export interface IChair {
      * Boolean value for whether someone is seated in the chair
      * True if someone is seated in the chair, otherwise false
      */
-    isSeated?: boolean;
+    isSeated: boolean;
+    /**
+     * Name of Person Sitting on Chair
+     */
+    occupiedBy: string;
+    /**
+     * Determines if the chair will be visible or not
+     */
+    isVisible: boolean;
+    /**
+     * Boolean value for whether the chair is round.
+     * True if the chair is round, otherwise false.
+     */
+    isRound?: boolean;
 }
 
-// Define a type for Position to restrict to four specific values
-type Position = 'top' | 'bottom' | 'left' | 'right';
-
 /**
  * Primary UI component for user interaction
  */
-export const Chair: React.FC<IChair> = ({ position = 'top', ...props }) => {
-    const { colors } = useTheme();
+export const Chair: React.FC<IChair> = ({
+    position = 'top',
+    isSeated = false,
+    occupiedBy = '',
+    isVisible = true,
+    isRound = false,
+    ...props
+}) => {
+    if (isRound) {
```

Comment:
Put this inside the return statement it looks out of place, 
Do
IsRound ?
GetCurvedCircleChair
:
GetStraightChair